new/LIT6199WeddaseMalkeaSellase.xml

```diff
@@ -0,0 +1,89 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6199WeddaseMalkeaSellase" xml:lang="en"
+    type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+
+                <title xml:lang="gez" xml:id="t1">ውዳሴ፡ ዘመልክአ፡ ሥላሴ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Wǝddāse zaMalkǝʾa
```

Comment:
zaMalkǝʾa is not a transcription style used until now in BM. I would write za-malkǝʾa, but if it is important to mark Malkǝʾa...as the title of another work, I would also understand the capital.